book/content/continuous_integration/continuous_integration.md

```diff
@@ -351,6 +351,8 @@ Because the build and test steps must be performed frequently, it is essential t
 
 When possible, running different sections of the test suite in parallel can help move the build through the pipeline faster. Care should also be taken to make sure the proportion of each type of test makes sense. Unit tests are typically very fast and have minimal maintenance overhead. In contrast, automated system or acceptance testing is often complex and prone to breakage. To account for this, it is often a good idea to rely heavily on unit tests, conduct a fair number of integration tests, and then back off on the number of later, more complex testing.
 
+Where CI steps are run sequentially, organise them so that the most important, or those that give the most valuable feedback, run first. Doing so ensures that if these steps fail the pipeline exits earlier, reducing the latency between pushing a change and receiving useful information.
```

Comment:
```suggestion
Where CI steps are run sequentially, organise them so that the most important, or those that give the most valuable feedback, run first. 
Doing so ensures that if these steps fail, the pipeline exits earlier that reduces the latency between pushing a change and receiving useful information.
```

Reply:
Again line break and minor formatting.